src/apps/reports/src/app/views/ActivityLog/utils.ts

```diff
@@ -0,0 +1,18 @@
+import moment from "moment";
+
+// This assumes the date passed in the param has a format of YYYY-MM-DD
+export const toUTC = (originalDate: string) => {
```

Comment:
If we only pass in the YYYY-MM-DD dateString moment defaults the time to be always 00:00:00 which makes the UTC conversion not work properly since we need it to base the UTC conversion off of the user's current local time.